### PR TITLE
drop -noroot for recent FLUENT versions, honor installopts, enable -debug

### PR DESCRIPTION
**Pull request by boegel**
@wpoely86: please review, well (re)test all existing FLUENT easyconfigs with this in place

### CODE REVIEW

File: easybuild/easyblocks/f/fluent.py [hunk 1]
```diff
@@ -55,7 +55,12 @@ def build_step(self):
     def install_step(self):
         """Custom install procedure for FLUENT."""
 
-        cmd = "./INSTALL -noroot -silent -install_dir %s" % self.installdir
+        extra_args =''
+        # only include -noroot flag for older versions
+        if LooseVersion(self.version) < LooseVersion('15.0'):
+            extra_args += '-noroot'
+
+        cmd = "./INSTALL %s -debug -silent -install_dir %s %s" % (extra_args, self.installdir, self.cfg['installopts'])
```
Comment (wpoely86) [inline easybuild/easyblocks/f/fluent.py]: we want debug enabled always?

Comment (boegel) [inline easybuild/easyblocks/f/fluent.py]: doesn't hurt, I guess

the real solution for large (debug) logs is supporting automatically zipping them at the end of the installation ;)

Comment (wpoely86) [inline easybuild/easyblocks/f/fluent.py]: debug refers too the install process or the actual application being installed?

Comment (boegel) [inline easybuild/easyblocks/f/fluent.py]: the install process

Comment (wpoely86) [inline easybuild/easyblocks/f/fluent.py]: forget the remark than :wink:

**Comment by boegel** [comment]
retested with all existing FLUENT easyconfigs, nothing was broken, so going in

Thanks for the review @wpoely86!

**Pull request merged**